src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/GetResourceVersions.sql

```diff
@@ -0,0 +1,39 @@
+﻿--DROP PROCEDURE dbo.GetResourceVersions
+GO
+CREATE PROCEDURE dbo.GetResourceVersions @ResourceDateKeys dbo.ResourceDateKeyList READONLY
+AS
+-- This stored procedure allows to identifiy version gap is available
+set nocount on
+DECLARE @st datetime = getUTCdate()
+       ,@SP varchar(100) = 'GetResourceVersions'
+       ,@Mode varchar(100) = 'Rows='+convert(varchar,(SELECT count(*) FROM @ResourceDateKeys))
+       ,@DummyTop bigint = 9223372036854775807
```

Comment:
What is this being used for? 
It looks like it is trying to force the query generator to make a specific plan?

Reply:
That is correct. It asks for optimization with one row in TVP.